.infrastructure/secret.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+ name: app-secrets
+ namespace: todoapp
+type: Opaque
+data:
+ SECRET_KEY: 5.2
```

Comment:
The `SECRET_KEY` value should be base64 encoded. Currently, it is set as a plain number `5.2`. Please encode the secret value in base64 format before placing it here.